model/src/main/java/org/mskcc/cmo/metadb/model/SampleMetadata.java

```diff
@@ -471,6 +471,14 @@ public void setRequestId(String requestId) {
     public void setCmoSampleIdFields(Map<String, String> cmoSampleIdFields) {
         this.cmoSampleIdFields = cmoSampleIdFields;
     }
+    
+    @Override
```

Comment:
I'm not sure what this method is for?

We have a method already in MetaDbSample that fetches the latest `SampleMetadata` in its member list. Is that what this was intended for?

Reply:
This method is used for collections.sort used to sort a list of SampleMetadata by importDate and in ascending order